EIPS/eip-tbd.md

```diff
@@ -0,0 +1,199 @@
+---
+eip: TBD
```

Comment:
```suggestion
eip: 7519
```
Assigning next sequential EIP/ERC number.
EIP/ERC numbering changed to sequential from 7500 and is no longer the PR number.

Please also update the filename.